Turbolinks.xcodeproj/project.pbxproj

```diff
@@ -462,7 +462,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Are we okay with deprecating support for users who are using iOS 13? Do we know offhand how many users will be impacted by this change?
Wondering what's our usual decision-making process when we need to drop support for old iOS versions. cc @steves99things @jpcamara